website/pages/docs/block.mdx

```diff
@@ -6,6 +6,22 @@ description: 'Million.js introduces the concept of a Block. Blocks are a way for
 import { Callout } from 'nextra-theme-docs';
 import { AutomaticModeWarning } from '../../components/automatic-mode-warning';
 
+## What's a block?
```

Comment:
Re: this page

You may want to determine how you want the structure of the reference docs to look like.

I'm not sure if you've finished this page yet or not, but there's a bit of repetition within it. It's also a bit different than how `<For>` is.

Reply:
Thanks for this, I resolved to this.

<img width="1241" alt="Screenshot 2023-12-07 at 13 39 47" src="https://github.com/aidenybai/million/assets/96334363/df0af49f-5d8d-4bf4-b3f7-082f900f1334">